scripts/upgrade

```diff
@@ -183,30 +183,16 @@ ynh_add_fpm_config --package="$extra_php_dependencies"
 # STORE THE CONFIG FILE CHECKSUM
 #=================================================
 
-### Verify the checksum of a file, stored by `ynh_store_file_checksum` in the install script.
-### And create a backup of this file if the checksum is different. So the file will be backed up if the admin had modified it.
-ynh_backup_if_checksum_is_different --file="$final_path/config/local.config.php"
-# Recalculate and store the checksum of the file for the next upgrade.
-ynh_store_file_checksum --file="$final_path/config/local.config.php"
-
-# Set up cron job
-ynh_replace_string --match_string="__YNH_WWW_PATH__" --replace_string="$final_path" --target_file="../conf/poller-cron"
-ynh_replace_string --match_string="__USER__" --replace_string="$app" --target_file="../conf/poller-cron"
-ynh_replace_string --match_string="__PHP_VERSION__" --replace_string="$phpversion"  --target_file="../conf/poller-cron"
-cp "../conf/poller-cron" "/etc/cron.d/$app"
+ynh_add_config --template="../conf/cron" --destination="/etc/cron.d/$app"
+chown root: "/etc/cron.d/$app"
+chmod 644 "/etc/cron.d/$app"
 
 # Run composer
 pushd "$final_path"
  	php$phpversion bin/console dbstructure update  2> /dev/null
```

Comment:
```suggestion
      php$phpversion bin/composer.phar install --no-dev
 	php$phpversion bin/console dbstructure update
```

Dependencies must be updated as well

Reply:
Dunno, if it is significant, but maybe we should use app specific version of php, but not use bash wrappers (since it uses just `php` without specific version):
```suggestion
 	php$phpversion bin/composer.phar install --no-dev
 	php$phpversion bin/console.php dbstructure update
```